lib/WebAppCommon/Util/EnsEMBL.pm

```diff
@@ -20,10 +21,21 @@ class_has registry => (
 
 sub _build_registry {
 
-    Bio::EnsEMBL::Registry->load_registry_from_db(
-        -host => $ENV{LIMS2_ENSEMBL_HOST} || 'ensembldb.internal.sanger.ac.uk',
-        -user => $ENV{LIMS2_ENSEMBL_USER} || 'anonymous',
-    );
+    my $public = 'ensembldb.ensembl.org';
+    my $host   = $ENV{LIMS2_ENSEMBL_HOST};
+    my $loaded;
+    try {
+        $loaded = Bio::EnsEMBL::Registry->load_registry_from_db(
+            -host => $host || $public,
+            -user => $ENV{LIMS2_ENSEMBL_USER} || 'anonymous',
+        );
+    };
+    if ( not $loaded and $host ) { 
```

Comment:
You can also just catch the error from trying to load public and internal Ensembl.

Reply:
Unfortunately that doesn't work, for some error conditions the EnsEMBL API throws an exception and for some others it just returns 0. By wrapping it in the _try_ we prevent the exception from bubbling up, so either way we wind up with a false value in $loaded if EnsEMBL couldn't be loaded.